packages/@react-native-windows/cli/src/test/dependencyConfig.test.ts

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
Depending on how long these takes, consider renaming `test` to `e2etest`. The difference is that unit tests are run in parallel, and on save when the cli project is open. So you would want to keep heavier stuff under e2etest.

I definitely need to document that somewhere...

Reply:
Fixed.